examples/directions/components/directions/directions.tsx

```diff
@@ -38,6 +39,10 @@ const DirectionsExample = () => {
       .catch((errorStatus: google.maps.DirectionsStatus) => {
         console.error(errorStatus);
       });
+
+    return () => {
+      directionsRenderer?.setMap(null);
```

Comment:
```suggestion
      if(directionsRenderer) {
        directionsRenderer.setMap(null);
      }
```
To keep it consistent with the rest? But also okay like this 👍  (If using please check the formatting, I messed it up :D)